appeals/web/src/server/lib/boolean-formatter.js

```diff
@@ -27,3 +29,17 @@ export function convertFromBooleanToYesNoWithOptionalDetails(boolean, optionalDe
 
 	return '';
 }
+
+/**
+ *
+ * @param {string | null | undefined} string
+ * @returns {Boolean}
+ */
+export function convertFromYesNoToBoolean(string) {
+	const lowerString = string?.toLowerCase();
+	if (lowerString !== 'yes' && lowerString !== 'no') {
+		logger.error('Not valid string');
+		throw new Error('Not valid string');
+	}
+	return lowerString === 'yes' ? true : false;
```

Comment:
I think you can just do `return lowerString === 'yes'` here - it will return the result of the check which is a boolean already, no need for the ternary